sphinxcontrib/doxylink/doxylink.py

```diff
@@ -385,8 +397,17 @@ def find_doxygen_link(name, rawtext, text, lineno, inliner, options={}, content=
             warning_messages.append('Could not find match for `%s` because tag file not found' % part)
             return [nodes.inline(title, title)], []
 
+        # Handle path:filename format
+        target_path = None
+        target_name = part
+        if ':' in part:
+            target_path, target_name = part.split(':', 1)
```

Comment:
Shouldn't we use [`rsplit`](https://docs.python.org/3/library/stdtypes.html#str.rsplit) instead? Here we use a colon as separator, but the example in the PR description uses a forward slash. What am I missing?